development.ini

```diff
@@ -27,24 +27,28 @@ jujugui.interactive_login = true
 
 # Default  URLS
 jujugui.jem_url =
```

Comment:
I'm leaving all the jem related code in and will remove it later on in a jem removal branch.